Source/WebKit/Shared/API/APIData.h

```diff
@@ -49,14 +48,14 @@ class Data : public ObjectImpl<API::Object::Type::Data> {
 
     static Ref<Data> create(std::span<const uint8_t> bytes)
     {
-        uint8_t* copiedBytes = nullptr;
+        std::span<uint8_t> copiedBytes;
 
-        if (bytes.size()) {
-            copiedBytes = static_cast<uint8_t*>(fastMalloc(bytes.size()));
-            memcpy(copiedBytes, bytes.data(), bytes.size());
+        if (!bytes.empty()) {
+            copiedBytes = unsafeForgeSpan(static_cast<uint8_t*>(fastMalloc(bytes.size())), bytes.size());
```

Comment:
I bet we can make a MallocSpan type, or just a fastMallocSpan helper that returns a span.

Reply:
Ok, in my next PR, I will look into introducing some sort of "owning" span which calls a deleter in its destructor.